platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/main/java/com/swirlds/platform/test/chatter/simulator/EventTracker.java

```diff
@@ -163,12 +155,10 @@ public void purge(final long olderThanRound) {
     /**
      * Capture event statistics on an event that is about to be purged.
      *
-     * @param descriptor
-     * 		a description of the event
-     * @param event
-     * 		the destination event
+     * @param descriptor a description of the event
+     * @param event      the destination event
      */
-    private void captureStatistics(final ChatterEventDescriptor descriptor, final TrackedEvent event) {
+    private void captureStatistics(final EventDescriptor descriptor, final TrackedEvent event) {
```

Comment:
Null annotations

Reply:
I am not going to add those here, since I only changed the type.